RequestService/RequestService.Handlers/BusinessLogic/MultiJobs.cs

```diff
@@ -34,7 +34,9 @@ public void AddMultiVolunteers(NewJobsRequest request)
                             Questions = j.Questions,                            
                             NumberOfRepeats = j.NumberOfRepeats,
                             RepeatFrequency = j.RepeatFrequency,
-                            NotBeforeDate = j.NotBeforeDate
+                            NotBeforeDate = j.NotBeforeDate,
+                            Multi = true,
```

Comment:
I think the Multi and Repeat flags should apply to the initial jobs as well, not just the extra ones.  (Together they make a set of multi/repeated jobs.)